### PR TITLE
Swiss gateway integration

### PR DESCRIPTION
**Pull request by hilmarf**
upload + download enhancements

### CODE REVIEW

File: common/persistence/src/main/resources/db/migration/V9__createAndUpdateTablesForSwissFederation.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE chgs_upload_key (
```
Comment (hilmarf) [inline common/persistence/src/main/resources/db/migration/V9__createAndUpdateTablesForSwissFederation.sql:1]: can't we combine this with `V8__alterFederationBatchInfoAddSource.sql` ?

File: common/persistence/src/main/resources/db/specific/postgresql/V10__createTriggersAndPermissionsForSwissFederation.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+CREATE TRIGGER remove_expired_uploadable_swiss_keys_trigger
+    AFTER DELETE ON diagnosis_key
+    FOR EACH ROW EXECUTE PROCEDURE remove_expired_swiss_uploadable_keys();
+
+    
+-- Add the necessary permissions for the swiss federation gateway key table and replication triggers
+
+GRANT SELECT, DELETE ON TABLE chgs_upload_key TO "cwa_chgs_upload";
+GRANT SELECT, DELETE ON TABLE chgs_upload_key TO "cwa_distribution";
+GRANT INSERT ON TABLE chgs_upload_key TO "cwa_submission";
```
Comment (hilmarf) [inline common/persistence/src/main/resources/db/specific/postgresql/V10__createTriggersAndPermissionsForSwissFederation.sql:36]: is this collected in a run-book? does TSI ops know how to setup new user & role?

Comment (hilmarf) [reply common/persistence/src/main/resources/db/specific/postgresql/V10__createTriggersAndPermissionsForSwissFederation.sql:36]: @EugenM-SAP - also circleCI is failing with `Caused by: org.postgresql.util.PSQLException: ERROR: role "cwa_chgs_upload" does not exist`

**Review by FelixRottler — APPROVED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -92,7 +92,7 @@ services:
     ports:
       - "8011:8080"
     environment:
-      SPRING_PROFILES_ACTIVE: debug,disable-ssl-client-postgres
+      SPRING_PROFILES_ACTIVE: debug,disable-ssl-server,disable-ssl-client-postgres,disable-ssl-client-verification,disable-ssl-client-verification-verify-hostname,disable-ssl-efgs-verification
```
Comment (hilmarf) [inline docker-compose.yml:95]: why we need all those?

**Review by EugenM-SAP — APPROVED** [review]

**Pull request merged**